router/server.go

```diff
@@ -150,6 +152,8 @@ func NewRouter(ctx context.Context, h models.HandlerInterface, port int, g http.
 		Methods("DELETE")
 	gMux.Handle("/api/filter/file/{id}", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.GetMesheryFilterFileHandler)))).
 		Methods("GET")
+	gMux.Handle("/api/filter/{id}/fork", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.ForkMesheryFilterHandler)))).
```

Comment:
@gr455 Can we make endpoint as `/api/filter/fork/{id}`. This resembles other endpoints.